### PR TITLE
Newsletter 2022 07

### PR DESCRIPTION
**Pull request by vdauwera**
Last call for edits? 

@nlharris I think this is ready to go? Sorry for the radio silence in August, I was out for most of it.

### CODE REVIEW

File: newsletters/2022-07.md
Comment (nlharris) [inline newsletters/2022-07.md:21]: their presentation slides and/or posters

File: newsletters/2022-07.md
Comment (nlharris) [inline newsletters/2022-07.md:36]: Add link to blog post about call for applications for Oct 1 round? https://www.open-bio.org/2022/08/12/obf-event-fellowship-2022-round2/
(I'm not sure where best to add it)

**Review by nlharris — APPROVED** [review]
Looks good; I made two minor comments.

**Review by hlapp — APPROVED** [review]
See inline comments, but 👍

**Comment by nlharris** [comment]
I agree with Hilmar's suggest to make the link to the BOSC 2022 page rather than the BOSC page that changes every year.

As far as I'm aware, ISCB has not yet made the keynote videos available.

**Comment by nlharris** [comment]
Regarding the keynote videos, I just pinged ISCB and asked if they were ready yet, and Seth said they were getting close. I don’t think we should block on that (especially since the newsletter is dated July/August, it would be good to get it out before October!) but if that comes through before the newsletter is sent, I’ll let you know.

File: newsletters/2022-07.md
Comment (vdauwera) [reply newsletters/2022-07.md:36]: done, worked it in

File: newsletters/2022-07.md
Comment (vdauwera) [reply newsletters/2022-07.md]: done

File: newsletters/2022-07.md
Comment (vdauwera) [reply newsletters/2022-07.md]: Great point about the URL, done.

I made sure to spell out COSI in the first line so I don't think we need to spell it out again for the BOSC instance.

**Comment by nlharris** [comment]
The BOSC report is now published on F1000Research: https://f1000research.com/articles/11-1034

File: newsletters/2022-07.md
Comment (nlharris) [inline newsletters/2022-07.md:21]:
```suggestion
A [report about BOSC 2022](https://f1000research.com/articles/11-1034) has been published as part of the [BOSC collection on F1000](https://f1000research.com/collections/bosc). All BOSC presenters are encouraged to submit their presentation slides and/or posters to the collection by September 30, 2022.
```

Comment (nlharris) [reply newsletters/2022-07.md:21]: (I just learned about "suggesting mode" in PR reviews -- super-handy!

**Comment by vdauwera** [comment]
Ooh I like suggestion mode. 

Ok, I think this is good to go now, for real. Assuming it is, what is the next step? We merge the PR, then someone actually sends the newsletter? (Who and how exactly?)

**Comment by nlharris** [comment]
Yeah, let's give @peterjc a chance to review and merge, and Peter can also decide who should send it. (In the past, it was sent out by the newsletter editor, who was also an OBF Board member. This is the first time we've had a community volunteer put together the newsletter!)

**Comment by nlharris** [comment]
We have a Board hangout this Friday so if Peter hasn't looked at it by then I can bring it up.

File: newsletters/2022-07.md
Comment (peterjc) [inline newsletters/2022-07.md:9]: Maybe July/August/September now?

Or just say September and change the filename to match.

**Review by peterjc — APPROVED** [review]
Suggestion on title date, but either way best send this sooner rather than later.

**Comment by peterjc** [comment]
Added comment on #26 about sending out the emails.

File: newsletters/2022-07.md
Comment (peterjc) [reply newsletters/2022-07.md:9]:
```suggestion
# OBF Newsletter July/August/September 2022
```

Comment (nlharris) [reply newsletters/2022-07.md:9]: makes sense

**Comment by nlharris** [comment]
I don't have merge permission (why?); can someone who does merge this?

**Comment by hlapp** [comment]
> I don't have merge permission (why?)

You weren't part of the @OBF/newsletter team. I've added you to it now.

**Pull request merged**
